workbench/fastify/nitro.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineNitroConfig } from 'nitro/config';
+
+export default defineNitroConfig({
+  modules: ['workflow/nitro'],
+  vercel: { entryFormat: 'node' },
+  routes: {
+    '/api/**': './src/index.ts',
```

Comment:
need to do this so that nitro can handle other routes from fastify server

```suggestion
    '/**': './src/index.ts',
```